python/paddle/fluid/tests/unittests/test_buffer_shared_inplace_pass.py

```diff
@@ -171,6 +175,16 @@ def test_multi_card_fetch_var(self):
 class CPUInplaceTest(InplaceTestBase):
     def initParameter(self):
         self.use_cuda = False
+        self.fuse_all_optimizer_ops = False
+
+
+class CUDAInplaceTestWithFuseOptimizationOps(InplaceTestBase):
+    def initParameter(self):
+        self.use_cuda = True
+        # FIXME(zcd): if training is in CPU, fuse_all_optimizer_ops
```

Comment:
Why? There should be no difference between CPU and GPU... And you mean, we can't set this option to `True` when training on CPU?

Reply:
>  if training is in CPU, fuse_all_optimizer_ops may cause some diff with the original model, the difference is about 1e-8. This may be related to byte alignment.